src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -321,6 +359,32 @@ private HttpServerOptions httpServerOptions() {
         HttpServerOptions httpServerOptions = new HttpServerOptions();
         httpServerOptions.setHost(this.bridgeConfig.getHttpConfig().getHost());
         httpServerOptions.setPort(this.bridgeConfig.getHttpConfig().getPort());
+
+        if (this.bridgeConfig.getHttpConfig().isSslEnabled()) {
+            httpServerOptions.setSsl(true);
+
+            if (bridgeConfig.getHttpConfig().getHttpServerSslKeystoreLocation() != null && this.bridgeConfig.getHttpConfig().getHttpServerSslKeystoreKeyLocation() != null) {
+                httpServerOptions.setKeyCertOptions(new PemKeyCertOptions()
+                        .setKeyPath(this.bridgeConfig.getHttpConfig().getHttpServerSslKeystoreKeyLocation())
+                        .setCertPath(this.bridgeConfig.getHttpConfig().getHttpServerSslKeystoreLocation()));
+            }
+
+            if (bridgeConfig.getHttpConfig().getHttpServerSslKeystoreCertificateChain() != null && this.bridgeConfig.getHttpConfig().getHttpServerSslKeystoreKey() != null) {
+                httpServerOptions.setKeyCertOptions(new PemKeyCertOptions()
+                        .addKeyValue(Buffer.buffer(this.bridgeConfig.getHttpConfig().getHttpServerSslKeystoreKey()))
+                        .addCertValue(Buffer.buffer(this.bridgeConfig.getHttpConfig().getHttpServerSslKeystoreCertificateChain())));
+            }
```

Comment:
should we, at least, logging some warning if the user enables SSL but doesn't complete the configuration in terms of certificates and keys?